src/folio_migration_tools/mapping_file_transformation/ref_data_mapping.py

```diff
@@ -43,6 +43,8 @@ def get_ref_data_tuple(self, key_value):
         return self.cached_dict.get(key_value.lower().strip(), ())
 
     def setup_mappings(self):
+        if not self.map:
+            return
```

Comment:
If we're going to return None here, we should probably log the fact that the mapping is empty/doesn't exist
```suggestion
        if not self.map:
            logging.info("%s legacy map file is empty or not provided", self.name)
            return
```